public/src/components/server_config/bop.vue

```diff
@@ -104,7 +113,9 @@ export default {
         "carModel_label": "Car Model #",
         "ballast_label": "Ballast: 0 to 100kg max.",
 		"restrictor_label": "Motor Restrictor: 0 to 20% max.",
-		"add_entry_button": "Add BOP"
+		"add_entry_button": "Add BOP",

```

Comment:
Please, fix the identation (from tab to spaces)

Reply:
Done - thanks.